README.md

```diff
@@ -1,64 +1,95 @@
 # Ludwig documentation
 
-## Building docs locally
+Website: [ludwig.ai](ludwig.ai)
 
-Ludwig's documentation is build using [MkDocs](https://www.mkdocs.org/) and the beautiful [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) theme.
-In order to create Ludwig's documentation you have to install them:
+Ludwig's documentation is built using [MkDocs](https://www.mkdocs.org/) and the
+beautiful [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/)
+theme, deployed using [Mike](https://github.com/jimporter/mike).
 
-```
-pip install -r requirements.txt
-```
+## Edit-refresh development
 
-Generate `api.md` from source:
+1. Install requirements.
 
-```
-python code_doc_autogen.py
-```
+    ```
+    pip install -r requirements.txt
+    ```
 
-Run the web server:
+2. In terminal, keep a window running:
 
-```
-mkdocs serve
-```
+    ```
+    mkdocs serve
+    ```
 
-Navigate to <http://localhost:8000> to view the docs.
+3. Navigate to <http://localhost:8000> to view your changes.
 
-### Test versioning locally
+> :bulb: As you change markdown files in your text editor, you can simply refresh the page to see changes reflected.
```

Comment:
I actually think that everytime you save your changes using ctrl+s, it does a hot refresh and automatically updates

Reply:
Done.